league/dashboard/models.py

```diff
@@ -3,35 +3,97 @@
 
 from enum import Enum
 
-from league.database import Column, Model, SurrogatePK, db, reference_col
+from league.database import (Column, Model, SurrogatePK, association_proxy, db,
+                             reference_col, relationship)
 
 Color = Enum('Color', 'white black')
 
 
+class Player(SurrogatePK, Model):
```

Comment:
omg thank you. I had no idea the online-ratings project was also flask+sqlalchemy, awesome :)